CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to Stencil
```

Comment:
Do we need some kind of "Contributor License Agreement (CLA)" form going which the user has to fill before we accept the contribution ?